scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+RUN git clone https://github.com/sstephenson/bats.git \
+	&& (cd bats && ./install.sh /usr/local) \
+	&& rm -rf bats
+
+# Install python version
+
+# Install AWS CLI
+# RUN curl -O https://bootstrap.pypa.io/get-pip.py \
+# 	&& python3 get-pip.py --user \
+# 	&& pip3 install awscli --upgrade --user \
+# 	&& rm get-pip.py
+RUN pip install awscli --upgrade --user
```

Hadolint

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)